packages/dso-toolkit/package.json

```diff
@@ -45,11 +46,15 @@
     "highlight.js": "^10.7.2",
     "jquery": "^2.2.4",
     "jquery-pjax": "git+https://github.com/defunkt/jquery-pjax.git",
+    "node-sass": "^4.8.3",
     "prettier": "^2.2.1",
     "pretty": "2.0.0",
     "rollup": "^2.45.2",
     "stylelint": "13.12.0",
     "stylelint-config-standard": "20.0.0",
     "stylelint-scss": "3.19.0"
+  },
+  "resolutions": {
+    "node-sass": "4.14.1"
```

Comment:
Je zou ook met `sass` kunnen testen, liefst dat het het dan zonder die warnings doet.

Reply:
Heb ik overwogen, maar de CSS die wordt gepubliceerd wordt met `node-sass` gemaakt.

Reply:
Belangrijker lijkt me dat de SCSS wel ook met `sass` geconsumeerd wordt.

Reply:
Voor `dso-toolkit` is dat geen doelstelling (meer).

De bronbestanden uit `@dso-toolkit/sources` zullen in `@dso-toolkit/css` wel met `sass` worden gecompileerd.